tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -116,6 +119,24 @@ Spawn Notebook With Arguments
       Click Element  xpath://span[@id='jupyterhub-logo']
    END
 
+Spawn Notebook From Dashboard With Arguments
+  [Arguments]  ${img_displayed_name}  ${img_real_name}  ${retries}=1  ${size}=Small  ${spawner_timeout}=600 seconds  &{envs}
+  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  Menu.Navigate To Page    Applications    Enabled
+  Launch JupyterHub From RHODS Dashboard Dropdown
+  Login To Jupyterhub  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+  ${authorization_required} =  Is Service Account Authorization Required
+  Run Keyword If  ${authorization_required}  Authorize jupyterhub service account
+  Fix Spawner Status
+  Wait Until Page Contains Element  xpath://span[@id='jupyterhub-logo']
+  Wait Until Page Contains Element  xpath://input[@name="${img_displayed_name}"]
+  Wait Until Element Is Enabled    xpath://input[@name="${img_displayed_name}"]   timeout=600
```

Comment:
I think these two steps are superflous when using default images.
Minimal and SDS will be enabled from the beginning, while PyTorch and Tensorflow take more than 600 seconds to become enabled. I think the keyword should end on the previous line (Contains element jh-logo), and these lines and the next ones should instead be called from the TC if needed

Reply:
as agreed in an offline discussion, we agreed that the keyword should be more atomic than it actually was. So I reduced the scope of this keyword bringing the last lines of code in the TC. However, we agreed to include in the "Select Notebook Image" a wait condition for checking that the single image is select-able in the JH spawner before trying to spawn in.